common/src/main/java/com/wynntils/mc/event/ItemTooltipRenderEvent.java

```diff
@@ -52,8 +55,19 @@ public void setMouseY(int mouseY) {
 
     @Cancelable
     public static class Pre extends ItemTooltipRenderEvent {
-        public Pre(PoseStack poseStack, ItemStack itemStack, int mouseX, int mouseY) {
+        private List<Component> tooltips;
```

Comment:
it seems a bit strange to define this specific to the Pre event, but ultimately i suppose it doesn't really matter. if you really wanted to be pedantic, you could make all the variables/getters universal and define the setters in Pre

Reply:
I'm not even sure why we have a `Pre` event. We're only intercepting tooltip rendering once, yes? I only think we should need to bother with .Pre and .Post events if we *actually* are sending two different events. And I'm always willing to refactor, if it turns out that what was one event need to turn into two.